Flow.Launcher/Flow.Launcher.csproj

```diff
@@ -104,6 +104,9 @@
     <ProjectReference Include="..\Flow.Launcher.Plugin\Flow.Launcher.Plugin.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Resource Include="Images\gamemode.ico" />
```

Comment:
we dont need to do this separately because up top it's already including .ico files :)